### PR TITLE
Adding @aayush89890 profile

### PR DESCRIPTION
**Pull request by aayush89890**
Added my Profile (aayush89890) in the public/data/aayush89890.json

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/157"></a>

### CODE REVIEW

**Review by FayasNoushad — APPROVED** [review]

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]
Please add only one feature to the PR 👍

File: src/Components/Home.js [hunk 1]
```diff
@@ -24,9 +24,9 @@ function Home() {
   }, [])
 
   return (
-    <div>
+    <div className="alignCenter">
```
[the remark this answers is not in the document]
Comment (aayush89890) [reply src/Components/Home.js:27]: > Please add these changes in a seperate PR and link the issues the changes close please.
Always try to do only one thing in a PR, that makes it easier for the maintainer and you get more green squares (:

Thanks for your feedback...I am new to open source...will take care of this from next time.

[the remark this answers is not in the document]
Comment (aayush89890) [reply src/Components/Home.js:27]: @Cahllagerfeld can you help me in removing my other commits from this pull request?

[the remark this answers is not in the document]
Comment (schmelto) [reply src/Components/Home.js:27]: you can user `git revert <commit>` see [here](https://git-scm.com/docs/git-revert)

[the remark this answers is not in the document]
Comment (aayush89890) [reply src/Components/Home.js]: Hi @schmelto that helped.
Thank you

Can you please review my PR now, I have reverted all other changes.

File: public/data/aayush89890.json [hunk 2]
```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Aayush jain",
+    "bio": "Software Developer || YouTuber || Open Source Enthusiast",
+    "avatar": "https://avatars.githubusercontent.com/u/37421316?v=4",
```
Comment (Cahllagerfeld) [inline public/data/aayush89890.json:4]: Please use the link, specified in the readme, so the Avatar can autoupdate whenever you change it :D

**Review by Cahllagerfeld — CHANGES_REQUESTED** [review]
One small thing :D

File: public/data/aayush89890.json (hunk 2 above)
Comment (FayasNoushad) [reply public/data/aayush89890.json:4]: @aayush89890 Use `https://github.com/aayush89890.png`

Comment (aayush89890) [reply public/data/aayush89890.json]: @Cahllagerfeld Done!....Thanks for helping.

**Review by Cahllagerfeld — APPROVED** [review]
awesome! thanks for your contribution 💥

**Pull request merged**